README.md

```diff
@@ -15,13 +15,21 @@
 4. `cd` into the repo using Command Prompt/Terminal
 5. Run the following command:
      `vagrant up`
-5. This would be a good time to rewatch your favorite TV series installment, cook & have breakfast/lunch/dinner, walk the dog, clean your room, etc.<br>
-   By the time you get back the install will surely have completed.<sup>1</sup>
-6. ???
-7. Profit!
+6. Once vagrant exists with an error, execute `vagrant up` again (Workaround to install the linux headers required by the guest additions)
+7. This would be a good time to rewatch your favorite TV series installment, cook & have breakfast/lunch/dinner, walk the dog, clean your room, etc.<br>
+By the time you get back the install will surely have completed.<sup>1</sup>
+8. To confirm the installation worked, open the web browser of your choice and enter `http://192.168.64.78:3000/` into the address bar and see if the website loads correctly.
```

Comment:
This should probably not say a specific a local IP address, might be best to assume folks are installing on the device they're viewing from (however unlikely that is in any actual setup) and put `http://127.0.0.1:3000/` here?

Reply:
Do people really use the Debian system that is set up by the vagrant file, if so, I would agree with you, but I believe most people would probably access the website using the host machine of the Debian VM.
But maybe that's just me, because I love Windows :P
What do you suggest, change it to `http://127.0.0.1:3000/`, leave it like it is or just remove that step?

Reply:
You know what, I'm an idiot, and missed that the vagrantfile does specify the internal IP to use. Sorry about that! 
Feel free to close this comment then >_<

Reply:
No Problem